src/Controls/src/Core/HandlerImpl/Label/Label.Android.cs

```diff
@@ -76,13 +76,12 @@ void OnLayoutChanged(object sender, LayoutChangedEventArgs args)
 				var platformView = labelHandler.PlatformView;
 				var virtualView = labelHandler.VirtualView as Label;
 
-				if (platformView == null || virtualView == null)
+				if (platformView is null || virtualView is null)
 					return;
 
-				SpannableString spannableString = null;
-
-				if (virtualView.FormattedText != null)
-					spannableString = virtualView.ToSpannableString();
+				FormattedString formattedText = virtualView.FormattedText ?? virtualView.Text;
```

Comment:
My PR that is currently causing the merge conflict on this one fixed the issue where the measure was happening for plain text labels. Not sure if this is still needed.

Reply:
Checked and resolved the conflict.